config/routes.rb

```diff
@@ -12,13 +12,25 @@
       get :current_graph
     end
   end
+
+  namespace :admin do
+    get '/', to: 'panel#index'
+  end
+
   resources :dashboard, only: [:index]
-  resources :assesments, only: [:index, :create]
+
+  resources :users, only: [:index]
+
+  resources :assesments, only: [:index, :create] do
+    collection do
+      get :all_assesments
```

Comment:
this action has to reside in a different name space that only belong to an admin. i.e is there will be a different set of admin controllers here